Dashboard/app/js/lib/controllers/survey-controllers.js

```diff
@@ -569,6 +569,28 @@ FLOW.projectControl = Ember.ArrayController.create({
       var groups = FLOW.router.approvalGroupListController.get('content');
       return groups && groups.filterProperty('keyId', approvalGroupId).get('firstObject');
   }.property('this.currentProject.dataApprovalGroupId'),
+
+  saveProject: function() {
+    var currentProject = this.get('currentProject');
+    var currentForm = this.get('selectedSurvey');
```

Comment:
This one ends up undefined. For now lets keep it picking from ` FLOW.selectedControl.get('selectedSurvey')` till we sort out the state issues.